source/Government.cpp

```diff
@@ -148,6 +153,10 @@ void Government::Load(const DataNode &node)
 			child.PrintTrace("Error: Expected key to have a value:");
 		else if(key == "player reputation")
 			initialPlayerReputation = add ? initialPlayerReputation + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(key == "reputation max")
+			reputationMax = child.Value(valueIndex);
+		else if(key == "reputation min")
+			reputationMin = child.Value(valueIndex);
```

Comment:
```suggestion
		else if(key == "reputation max")
			reputationMax = add ? reputationMax + child.Value(valueIndex) : child.Value(valueIndex);
		else if(key == "reputation min")
			reputationMin = add ? reputationMin + child.Value(valueIndex) : child.Value(valueIndex);
```
The example is just above btw, with player reputation.